gem/lib/pagy/i18n.rb

```diff
@@ -113,6 +113,7 @@ module P11n
                  hash['zh-CN'] = RULE[:other]
                  hash['zh-HK'] = RULE[:other]
                  hash['zh-TW'] = RULE[:other]
+                 hash['dz']    = RULE[:other]
```

Comment:
This line is inconsistent with the pluralization indicated in the `gem/locales/dz.yml` and its entries.

Reply:
thank you, will update